maxim/logger/crewai/client.py

```
@@ -53,6 +54,18 @@ def get_log_level(debug: bool) -> int:
     return logging.DEBUG if debug else logging.WARNING
 
+class MaximEvalConfig:
+    """Maxim eval config.
+
+    This class represents a maxim eval config.
+    """
+
+    evaluators: list[str]
+    additional_variables: list[dict[str, str]]
+
+    def __init__(self):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)